app/notification/application/map_contents.py

```diff
@@ -27,9 +27,12 @@ class Application:
     @property
     def format_submission_date(self):
         if self.submission_date is not None:
-            return datetime.strptime(
-                self.submission_date, "%Y-%m-%dT%H:%M:%S.%f"
-            ).strftime("%Y-%m-%d")
+            return (
+                datetime.strptime(self.submission_date, "%Y-%m-%dT%H:%M:%S.%f")
```

Comment:
Can we set timezone to UK? Else this time will be in UTC.

Reply:
wouldn't be better if it was set in the application store ?